bin/hammer

```diff
@@ -31,6 +31,11 @@ class PreParser < Clamp::Command
   option ["--output"], "ADAPTER", "Set output format"
   option ["--csv-separator"], "SEPARATOR", "Character to separate the values"
   option ["--autocomplete"], "LINE", "Get list of possible endings"
+  option ["--verify_ssl"], "VERIFY_SSL", "Configure SSL verification of remote system"
+  option ["--ssl_ca_file"], "CA_FILE", _("configure the file containing the CA certificates")
+  option ["--ssl_ca_path"], "CA_PATH", _("configure the directory containing the CA certificates")
+  option ["--ssl_client_cert"], "CERT_FILE", _("configure the client's public certificate")
+  option ["--ssl_client_key"], "KEY_FILE", _("configure the client's private key")
```

Comment:
Can you please replace underscores with hyphens in the options? Hammer doesn't use underscores anywhere else in parameters.
`--ssl_client_key` -> `--ssl-client-key`